docs/sections/user_guide/cli/drivers/cdeps/index.rst

```diff
@@ -1,23 +1,11 @@
 ``cdeps``
 =========
 
-.. include:: /shared/idempotent.rst
+.. include:: ../shared/idempotent.rst
```

Comment:
We can use a relative path here because `idempotent.rst` doesn't transitively `include` anything that it can't locate.